libs/utils/android/system.py

```diff
@@ -464,4 +465,31 @@ def packages_info(target, apk_filter=''):
             return packages
         return None
 
+    @staticmethod
+    def grant_permissions(target, package):
```

Comment:
Nightly thought: this should probably be moved to devlib's android utilities to lessen the work involved in moving all of the System/Screen stuff to devlib (as mentionned in #469